README.md

```diff
@@ -0,0 +1,58 @@
+# EdgeCast CLI
+
+This is the official command line interface (CLI) for EdgeCast.
+
+## Getting Started
+
+### Requirements
+- Go 1.17
+
+### Installation
+To install the CLI, use the makefile:
```

Comment:
I think we should take how edge-compute team did. Basically we should have customers to download installation script and run it in order to install the cli and needed config files as necessary